branchLayoutApi/src/main/java/com/virtuslab/branchlayout/api/manager/IBranchLayoutReader.java

```diff
@@ -0,0 +1,8 @@
+package com.virtuslab.branchlayout.api.manager;
+
+import com.virtuslab.branchlayout.api.BranchLayoutException;
+import com.virtuslab.branchlayout.api.IBranchLayout;
+
+public interface IBranchLayoutReader extends IIndentDefining {
```

Comment:
As mentioned in `IIndentDefining`, I don't think `IIndentDefining` should be exposed in the interface (plus even in the implemenation, it should be rather contained by the reader/saver than implemented/extended by them)

Reply:
`IIndentDefining` removed from api